lego/apps/events/views.py

```diff
@@ -297,6 +299,19 @@ def upcoming(self, request):
         serializer = self.get_serializer(queryset, many=True)
         return Response(serializer.data)
 
+    @decorators.action(
+        detail=False,
+        serializer_class=ImageGallerySerializer,
+        permission_classes=[permissions.IsAuthenticated],
```

Comment:
Hmm, I think we should discuss the permissions here. Might be a good idea to use event create permissions for this?